index.html

```diff
@@ -64,5 +64,211 @@ <h3>LET'S CONNECT</h3>
           </div>
       </div>
     </section>
+
+    <!--  End of  Intro Page ---------------------------------------------------------- -->
+
+    <section>
+      <div class="works-wrapper">
+        <div class="works-section">
+            <div class="work-snapshot">
+              <img src="assets/works-snapshot/Snapshoot Portfolio 1.png" alt="work-snapshot-1">
+            </div>
+            <div class="work-description">
+              <div class="desc-wrapper">
+                <h2>Tonic</h2>
+                <div class="dec-details">
+                  <h4>CANOPY</h4>
+                  <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot">
+                  <h4 class="grey-text">Back End Dev</h4>
+                  <img class="mobile-dot" src="assets/mobile/dot.png" alt="dot">
+                  <h4 class="grey-text">2015</h4>
+                </div>
+                  <p class="desc-text">A daily selection of privately <br> personalized reads; no accounts or <br> sign-ups required.</p>
+                  <div class="work-tech">
+                      <ul>
+                        <li>html</li>
+                        <li>css</li>
+                        <li>javascript</li>
+                      </ul>
+                  </div>
+              </div>
+            </div>
+            <div class="see-work">
+              <a href="#">See Project</a>
```

Comment:
- [ ] Good job for using the `a` tag for button but  As stated the project requirement you should Implement the button interactions (enable, hover, etc.). So Kindly Implement it for all the button under "Work" and "About " section.
 
![Capture](https://user-images.githubusercontent.com/93598189/205585648-5abbf491-2e45-447e-afc0-a14aa50cfb43.PNG)